### PR TITLE
added docker compose

### PR DESCRIPTION
**Pull request by Andywkh**
## Description
adding docker-compose component to ease visibility

## Rationale
To reduce complexity of spinning up the testnet env

## Example
Automated process on spinning up testnet env

## Changes
added docker-compose and its respective ReadME

### CODE REVIEW

**Review by j75689 — APPROVED** [review]

File: deployment/docker/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+$ docker-compose up
+```
+
+---
+**Tips** 
+
+```bash
+# Assigning permissions to config and data folders.
+$ chmod 777 -R ~/.gnfd/config
+$ chmod 777 -R ~/.gnfd/data
```
Comment (KeefeL) [inline deployment/docker/README.md:53]: Why not do this in the Step1 directly?

File: deployment/docker/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## Minimum System Requirements
+
+- 1 TB of free disk space, accessible at a minimum read/write speed of 100 MB/s.
+- 4 cores of CPU and 8 GB of memory (RAM).
+- A broadband Internet connection with upload/download speeds of at least 1 MB/s.
+
+## Setting up a working envrionment
+
+| Please check the [greenfield repo](https://github.com/bnb-chain/greenfield) for information on the testnet, including the correct version of the binaries to use and details about the config file. |
```
Comment (KeefeL) [inline deployment/docker/README.md:15]: Please check the [Release. Note](https://greenfield.bnbchain.org/docs/release-note/) for information on the testnet, and find details in [Greenfield Repo](https://github.com/bnb-chain/greenfield), ....

File: deployment/docker/README.md (hunk 1 above)
Comment (Andywkh) [reply deployment/docker/README.md]: fixed in the new commit

File: deployment/docker/README.md (hunk 2 above)
Comment (Andywkh) [reply deployment/docker/README.md]: fixed in the new commit

**Comment by KeefeL** [comment]
Can you update the PR description? we should follow the PR Template.

**Review by KeefeL — APPROVED** [review]

**Pull request merged**